script/openqa-clone-custom-git-refspec

```diff
@@ -18,19 +18,28 @@ if [ -z ${host+x} ] || [ -z ${job+x} ]; then
     job=${job_url##*/}
 fi
 
+function throw_json_error {
+    echo "in contents queried from $json_url:"
+    echo $json_data
```

Comment:
I think it'd be nicer to avoid the global `$json_data` here and pass it as an argument to the function instead.

Reply:
I actually wanted to avoid passing the arguments each time the function is called (it is called quite often). Since all of the code still fits nicely on the screen I double anybody has problems to see what's going on.

Reply:
Why would you void it?

Reply:
Because it adds more noise to the code than necessary. Let's @okurz decide. After all it is his script :-)

Reply:
In the last rebase I made it use arguments.